### PR TITLE
add some more pages to sitemap

### LINKED ISSUE #16905: add all (remaining) frontend pages to sitemap
follow up for #16508

addons-server doesn't "know about" every frontend page so the pr for #16508 just added the ones we already had defined in urls.pys (the vast majority of the urls in addons-server are behind auth, or apis).  

~Pages with pagination/filters are a problem - e.g. category pages - because we a) don't know how many pages there are without either attempting to replicate the same query that the API would run in elasticsearch, or actually calling the API, and b) filters cause a potential explosion in the number of pages (if the default filter is effectively filtering nothing out I think we can avoid the filtered results).~

~As we exclude search result this wasn't previously a concern, but category pages are effectively search pages and inherit the same difficulties that caused us to exclude them.~
(spun out category pages to #16959)

Also, a future problem will be keeping everything in sync on addons-server when pages are added/removed to addons-frontend.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #16905 - this turned out to be a small patch after the category result pages were spun out to a different issue.

### CODE REVIEW

File: src/olympia/amo/sitemap.py [hunk 1]
```diff
@@ -75,6 +83,12 @@ def location(self, item):
     'amo': AMOSitemap(),
     'addons': AddonSitemap(),
     'collections': CollectionSitemap(),
+    # category pages (wrapped search results of addons in a category)
+    # blocked-addons?
+    # user profiles
+    # error pages?
+    # addon review pages (+ direct?)
+    # addon privacy, eula, license, versions
```
Comment (willdurand) [inline src/olympia/amo/sitemap.py:91]: These comments look like your own todo list :p Do we really need to commit them?

**Review by willdurand — COMMENTED** [review]
So there is no test cases?

**Comment by eviljeff** [comment]
> 
> 
> So there is no test cases?

No additional test cases, no.  `test_amo_sitemap` already trivially checks all the items and testing that `reverse` works is beyond the scope of the patch.

File: src/olympia/amo/sitemap.py (hunk 1 above)
Comment (eviljeff) [reply src/olympia/amo/sitemap.py:91]: heh, maybe?  There's a splattering of `# TODO:` all over this file.  But I'll remove.

**Comment by willdurand** [comment]
> No additional test cases, no. `test_amo_sitemap` already trivially checks all the items and testing that `reverse` works is beyond the scope of the patch.

ah ok, makes sense.

**Review by willdurand — APPROVED** [review]
r+wc

My only comment is about the comments (hehe).

**Pull request merged**